### DISCUSSION TITLE
[QUESTION]  ChannelInfo cachingPeriod typical value and maximum value

### DISCUSSION THREAD

**Discussion by wei-zhang-simplisafe** [awslabs/amazon-kinesis-video-streams-webrtc-sdk-c#1656] [Q&A] [answered]
Have questions about cachingPeriod setting.
The default cachingPeriod is set as 10 hrs.
Wonder if we can set it longer, such as 24 hrs?  What is the maximum it can go?
When will the Channel Endpoints expire/change on server side?
When the channel endpoints expire/change, what will happen if webrtc uses old endpoints to try lws connections?

**Comment by wei-zhang-simplisafe** [comment]
Any comments?

**Comment by disa6302** [comment, marked as answer]
There isnt a max value. You could cache it for even 24 hours. If the endpoint expires, the Signaling client will go through the right state transition to re-acquire the fresh values and update the cache.

**Reply by wei-zhang-simplisafe** [reply]
Thank you!
